index.thread.js

```diff
@@ -1,6 +1,8 @@
 // @flow
 import { self } from 'react-native-threads';
-import { HDNode } from 'ethers';
+import { utils } from 'ethers';
+
+const { HDNode } = utils;
```

Comment:
Note: the bundles should be updated on the ios/android app
yarn build-thread-ios
yarn build-thread-android